webapp/static/interop-data.json

```diff
@@ -322,119 +322,118 @@
                 "description": "Cascade Layers",
                 "mdn": "https://developer.mozilla.org/docs/Web/CSS/@layer",
                 "spec": "https://drafts.csswg.org/css-cascade/#layering",
-                "tests": "/results/css/css-cascade?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-cascade", 
+                "tests": "/results/css/css-cascade?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-cascade",
                 "countsTowardScore": false
             },
             "interop-2022-dialog": {
                 "description": "Dialog Element",
                 "mdn": "https://developer.mozilla.org/docs/Web/HTML/Element/dialog",
                 "spec": "https://html.spec.whatwg.org/multipage/interactive-elements.html#the-dialog-element",
-                "tests": "/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-dialog", 
+                "tests": "/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-dialog",
                 "countsTowardScore": false
             },
             "interop-2022-text": {
                 "description": "Typography and Encodings",
                 "mdn": "",
                 "spec": "",
-                "tests": "/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-text", 
+                "tests": "/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-text",
                 "countsTowardScore": false
             },
             "interop-2022-viewport": {
                 "description": "Viewport Units",
                 "mdn": "",
```

Comment:
https://developer.mozilla.org/docs/Web/CSS/length#relative_length_units_based_on_viewport could be a good link

Reply:
added